hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MemStoreChunkPool.java

```diff
@@ -77,6 +77,7 @@
   private static final int statThreadPeriod = 60 * 5;
   private AtomicLong createdChunkCount = new AtomicLong();
   private AtomicLong reusedChunkCount = new AtomicLong();
+  private AtomicLong gotChunkCount = new AtomicLong();
```

Comment:
Super minor nit.
"has" instead of "got" is more common, in our code base and elsewhere

Reply:
It's a counter for the number of method `Chunk#getChunk` get called. Agree, "got" is not good. What about `requestedChunkCount`.